deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -358,10 +501,15 @@ services:
       GRAPH_HTTP_ADDR: 0.0.0.0:9120
       GRAPH_SPACES_WEBDAV_BASE: https://${OCIS_DOMAIN:-ocis.owncloud.test}
 
-      REVA_GATEWAY: storage-gateway:9142
+      GRAPH_LDAP_URI: ldaps://localhost:9235
```

Comment:
how could this work?

Reply:
well exposed port => docker, but I fixed this to `ldaps://idm:9235`